openlibrary/templates/lib/nav_foot.html

```diff
@@ -38,7 +38,7 @@ <h2>$:_('Develop')</h2>
       <div>
         <h2>$:_('Help')</h2>
         <ul>
-          <li><a href="/help" title="$_('Help Center')">$_('Help Center')</a></li>
+          <li><a href="/help" title="$_('Help & Support')">$_('Help & Support')</a></li>
```

Comment:
this should be done in a separate change. This is scope creep and that makes it harder for deployers if they need to revert functionality.

Reply:
Okay.